### PR TITLE
Add request received event (customReceivedMessage)

### PR DESCRIPTION
**Pull request by nicolaichuk**
Change:

* add request received event (customReceivedMessage)
* add yarn.lock to .gitignore
* add t.end() to some test

Please review and merge to upstream.

### CODE REVIEW

**Review by jsumners — APPROVED** [review]
Looks good to me.

**Review by mcollina — APPROVED** [review]
lgtm

File: logger.js [hunk 1]
```diff
@@ -133,6 +134,9 @@ function pinoLogger (opts, stream) {
       }
 
       if (shouldLogSuccess) {
+        const level = customLogLevel ? customLogLevel(res) : useLevel
```
Comment (mcollina) [inline logger.js:137]: Could you add another check if `level` is undefined? In this case it should not log the next line.

File: logger.js [hunk 2]
```diff
@@ -54,6 +54,7 @@ function pinoLogger (opts, stream) {
   const autoLoggingGetPath = opts.autoLogging && opts.autoLogging.getPath ? opts.autoLogging.getPath : null
   delete opts.autoLogging
 
+  const receivedMessage = opts.customReceivedMessage || function () { return 'request received' }
```
Comment (mcollina) [inline logger.js:57]: I would prefer to have pino-http not log this message by default.

**Review by mcollina — CHANGES_REQUESTED** [review]

File: logger.js (hunk 2 above)
Comment (nicolaichuk) [reply logger.js]: Fixed

File: logger.js (hunk 1 above)
Comment (nicolaichuk) [reply logger.js]: Fixed

**Comment by nicolaichuk** [comment]
@mcollina, please review change.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**